### PR TITLE
Remove Unnecessary Tests in EditCommandTest #254

### PR DESCRIPTION
**Pull request by Zhiyuan-Amos**
Part of #254

### CODE REVIEW

**Comment by CanIHasReview** [comment]
# v1

@Zhiyuan-Amos submitted v1 for review.

* [v1 1/1] [Remove Duplicate Tests in EditCommandTest](https://github.com/se-edu/addressbook-level4/pull/395/commits/77fc9438ba6188b20a2e256d4616a9d19ccbf6ff)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/dc9fe96a38a2b5f9bb5bb4c05c3ff1db47b1effd...77fc9438ba6188b20a2e256d4616a9d19ccbf6ff))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/395/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by Zhiyuan-Amos** [comment]
ping @yamgent in case you forgot about this PR cos there's too many PRs going on :P

**Comment by yamgent** [comment]
In the commit message, you might want to specify which tests in the integration test are substituting the ones that will be deleted, so that readers will be able to verify whether it justifies removing then.

**Comment by CanIHasReview** [comment]
# v2

@Zhiyuan-Amos submitted v2 for review.

* [v2 1/1] [Remove Duplicate Tests in EditCommandTest](https://github.com/se-edu/addressbook-level4/pull/395/commits/34a1022e683b1f1d68c069d2ae419e58dc9762a1)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/a5d9fcc4cce6b798663a565091e38782d140b7e7...34a1022e683b1f1d68c069d2ae419e58dc9762a1)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level4/compare/407714553fa60645d3c95f7d47a30dae887f4761~1...407714553fa60645d3c95f7d47a30dae887f4761))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/395/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by damithc** [comment]
Let's put this on hold until EditCommandParserTest and EditCommandTest are merged.

File: src/test/java/guitests/EditCommandTest.java [hunk 1]
```diff
@@ -1,143 +1,68 @@
...
      * @param addressBookIndex index of person to edit in the address book.
-     *      Must refer to the same person as {@code filteredPersonListIndex}
      * @param detailsToEdit details to edit the person with as input to the edit command
      * @param editedPerson the expected person after editing the person's details
      */
-    private void assertEditSuccess(int filteredPersonListIndex, int addressBookIndex,
-                                    String detailsToEdit, Person editedPerson) {
-        commandBox.runCommand(EditCommand.COMMAND_WORD + " " + filteredPersonListIndex + " " + detailsToEdit);
+    private void assertEditSuccess(int addressBookIndex, String detailsToEdit, Person editedPerson) {
+        commandBox.runCommand("edit " + addressBookIndex + " " + detailsToEdit);
```
Comment (yamgent) [inline src/test/java/guitests/EditCommandTest.java]: Was this merged wrongly? I think it should be using `EditCommand.COMMAND_WORD`.

**Pull request closed without merging**